curvefs/test/metaserver/copyset/meta_operator_test.cpp

```diff
@@ -25,9 +25,9 @@
 #include <brpc/server.h>
 #include <gtest/gtest.h>
 
-#include <condition_variable>
-#include <mutex>
-#include <regex>
+#include <condition_variable>  // NOLINT
```

Comment:
you can add `--filter=-build/c++11` to cpplint to disable these false positive warnings.